### DISCUSSION TITLE
How to make a widget collapsible?

### DISCUSSION THREAD

**Discussion by cleargoal** [filamentphp/filament#15163] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

I created a custom Dashboard page in the user area and filled it with the default StatsOverview widget.
This widget contains 10 `Stat::make()` blocks with different project data for a user.
Now it looks pretty large in height and I want to make this widget collapsible.

I found something I assume I need in the documentation [here](https://filamentphp.com/docs/3.x/support/blade-components/section#making-a-section-collapsed-by-default).

```
<x-filament::section
    collapsible
    collapsed
>
    <x-slot name="heading">
        User details
    </x-slot>
 
    {{-- Content --}}
</x-filament::section>
```

I put such code into the component that bound to my Dashboard page. But I don't understand how to pass a widget to slot.

**Can you help me, please?**

**Comment by leandrocfe** [comment, marked as answer]
```html
<x-filament::section
    collapsible
>
    <x-slot name="heading">
        User details
    </x-slot>

    <div>
        @livewire(App\Filament\Widgets\Widget1::class)
        @livewire(App\Filament\Widgets\Widget2::class)
    </div>

</x-filament::section>
```

**Reply by cleargoal** [reply]
Thank you very much!
It works!
